src/emailpro/disclaimer/add/emailpro-disclaimer-add.html

```diff
@@ -29,7 +29,7 @@
                         <label for="variableSelect" class="oui-field__label oui-label"
                                data-translate="emailpro_ACTION_disclaimer_insert"></label>
                         <div class="oui-field__content">
-                            <div class="oui-select">
+                            <div class="oui-select mb-0 mr-3">
```

Comment:
Otherwise as you are on a flex container, you can apply `align-self-md-start` class to the sibling `<button>` element instead of dealing with multiple margins.

Reply:
This is done. Thanks a lot for the suggestion :)